ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -536,4 +538,218 @@ public void getCompletedJobs() throws IOException, SignatureException, XmlProces
+                    .putJobCreatedNotificationRegistrationSpectraS3(new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"));
+            notificationUUID = response.getJobCreatedNotificationRegistrationResult().getId();
+            assertThat(response.getStatusCode(), is(201));
+
+        } finally {
+            client.deleteJobCreatedNotificationRegistrationSpectraS3(
+                    new DeleteJobCreatedNotificationRegistrationSpectraS3Request(notificationUUID));
+        }
+    }
+
+    @Test
+    public void getJobCreatedNotification() throws IOException, SignatureException {
+        UUID notificationUUID = null;
+        try {
+            notificationUUID = client.putJobCreatedNotificationRegistrationSpectraS3(
+                    new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"))
+                    .getJobCreatedNotificationRegistrationResult().getId();
+
+            final GetJobCreatedNotificationRegistrationSpectraS3Response response = client
+                    .getJobCreatedNotificationRegistrationSpectraS3(new
+                            GetJobCreatedNotificationRegistrationSpectraS3Request(notificationUUID));
+
+            assertThat(response.getStatusCode(), is(200));
+
+        } finally {
+            client.deleteJobCreatedNotificationRegistrationSpectraS3(
+                    new DeleteJobCreatedNotificationRegistrationSpectraS3Request(notificationUUID));
+        }
+    }
+
+    @Test
+    public void deleteJobCreatedNotification() throws IOException, SignatureException {
+        final UUID notificationUUID = client.putJobCreatedNotificationRegistrationSpectraS3(
+                new PutJobCreatedNotificationRegistrationSpectraS3Request("test@test.test"))
+                .getJobCreatedNotificationRegistrationResult().getId();
+
+        final DeleteJobCreatedNotificationRegistrationSpectraS3Response response = client
+                .deleteJobCreatedNotificationRegistrationSpectraS3(new DeleteJobCreatedNotificationRegistrationSpectraS3Request(notificationUUID));
+
+        assertThat(response.getStatusCode(), is(204));
+    }
+
+    @Test
+    public void putJobCompletedNotification() throws IOException, SignatureException {
+        UUID notificationUUID = null;
+        try {
+            final PutJobCompletedNotificationRegistrationSpectraS3Response response = client
+                    .putJobCompletedNotificationRegistrationSpectraS3(
+                            new PutJobCompletedNotificationRegistrationSpectraS3Request("test@test.test"));
+            notificationUUID = response.getJobCompletedNotificationRegistrationResult().getId();
+            assertThat(response.getStatusCode(), is(201));
+        } finally {
+            client.deleteJobCompletedNotificationRegistrationSpectraS3(
+                    new DeleteJobCompletedNotificationRegistrationSpectraS3Request(notificationUUID));
```

Comment:
The test will already fail for the bad assert, that doesn't mean we also should cause an exception to be thrown.  Seems like it would make the failure confusing to diagnose.